manifests/init.pp

```diff
@@ -61,6 +63,8 @@
   $session_lastvalidation                                        = 'session.lastvalidation',
   $proxy_server                                                  = undef,
   $proxy_type                                                    = undef,
+  String $license                                                = '',
+  String $server_id                                              = '',
```

Comment:
I think these need to be Optional[String] and default to undef. Otherwise you will be breaking existing module users by overwriting conluence.cfg.xml unexpectedly (I don't think ```''``` equals false in Puppet4+ anymore)

Reply:
Done.